src/on_message.py

```diff
@@ -91,6 +96,60 @@ async def handle_rep(message):
                         member = message.guild.get_member(member)
                         await member.add_roles(role)
 
+def typing_timer(channel_id, user_id, ttl):
+    time.sleep(ttl)
+    if str(channel_id) not in channels_typing:
```

Comment:
remove the `str` from channel snowflakes and store them as `int` objects instead